.github/workflows/tag-releases.yaml

```
@@ -0,0 +1,34 @@
+# A workflow to create github releases with changelogs from tags
+
+name: 'Create github release for tag'
+
+on:
+  push:
+    tags:
+      - '*'
+workflow_dispatch:
```

Missing an indent here?

```suggestion
  workflow_dispatch:
```